cloud/sshshim/sshshim.go

```diff
@@ -0,0 +1,57 @@
+package sshshim
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"go.jetpack.io/devbox/boxcli/featureflag"
+	"go.jetpack.io/devbox/cloud/openssh"
+)
+
+const (
+	configShimDir = ".config/devbox/shims"
+)
+
+const sshShimContents = `#!/bin/sh
+# Next PR:
+# devbox cloud ssh "$@"
+ssh "$@"
```

Comment:
If you do keep the script, I'd make this an `exec` to replace the shell process.